dandi/tests/fixtures.py

```diff
@@ -63,6 +64,35 @@ def simple2_nwb(simple1_nwb_metadata, tmpdir_factory):
     )
 
 
+@pytest.fixture()
+def organized_nwb_dir(simple2_nwb, tmp_path, clirunner):
+    shutil.copy(str(simple2_nwb), str(tmp_path))
+    (tmp_path / dandiset_metadata_file).write_text("{}\n")
+    r = clirunner.invoke(organize, ["-f", "move", "--dandiset-path", str(tmp_path)])
+    assert r.exit_code == 0, r.stdout
+    return tmp_path
+
+
+@pytest.fixture()
+def organized_nwb_dir2(simple1_nwb_metadata, simple2_nwb, tmp_path, clirunner):
+    shutil.copy(str(simple2_nwb), str(tmp_path))
+    make_nwb_file(
+        str(tmp_path / "simple3.nwb"),
+        subject=pynwb.file.Subject(
+            subject_id="lizard001",
+            date_of_birth=datetime(2019, 12, 1, tzinfo=tzutc()),
+            sex="F",
+            species="Gekko gecko",
+        ),
+        **simple1_nwb_metadata,
+    )
+    (tmp_path / dandiset_metadata_file).write_text("{}\n")
+    r = clirunner.invoke(organize, ["-f", "move", "--dandiset-path", str(tmp_path)])
```

Comment:
Same here

Reply:
actually I was wrong here  - since one more file created straight in that path... so I just left it alone for now